assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -121,11 +124,27 @@ const OrderSummaryItem = ( { cartItem } ) => {
 				<ProductImage image={ images.length ? images[ 0 ] : {} } />
 			</div>
 			<div className="wc-block-components-order-summary-item__description">
+				<span className="screen-reader-text" tabIndex={ 0 }>
```

Comment:
We don't need to make these elements focusable, screen readers already offer a way to navigate unfocusable elements.

Reply:
Thanks for pointing that out, I've removed it.

Seems [VO key(CTRL+Option) + →] and [VO key + ←] would work for me as opposed to using the TAB key.